mrjones2014/yaclt
# Add support for user-defined entry file names

Resolves: #127 

## How to Test

1. Update `yacltrc.js` and add 
```js
entryFileName: (now) => `test-${now.toISO()}.md`
```
2. Running `yarn build; ./dist/index.js new --edit` should open a file with the name formatted like specified
3. Change
```js
entryFileName: (now) => `test-${now.toISO()}.md`
```
to
```js
entryFileName: (now) => `test.md`
```
4. Run `./dist/index.js new` twice -- it should fail the 2nd time because the file already exists
